### ISSUE TITLE
SideMenu addPanGestureToPresent problem version: 6.4.0

### ISSUE THREAD

**Issue by 22Round** [jonkykong/SideMenu#555]
**I have read the [guidelines for contributing](https://github.com/jonkykong/SideMenu/blob/master/.github/CONTRIBUTING.md) and I understand**
- [x] My issue is happening in the **latest version** of SideMenu (older versions are no longer maintained).
- [x] My issue was **not** solved in the [README](https://github.com/jonkykong/SideMenu/blob/master/README.md).
- [x] My issue can **not** be answered on [stackoverflow.com](stackoverflow.com).
- [x] My issue is **not** a request for new functionality that I am unwilling to build and contribute with a pull request.
- [x] My issue **is** reproducible in the [demo project](https://github.com/jonkykong/SideMenu/tree/master/Example).

**Describe the bug**
I updated SideMenu from 5.0.3 to 6.4.0 and this method crashes application
Environment: Xcode11; iOS 13

```
if let nav = self.navigationController {
    SideMenuManager.default.addPanGestureToPresent(toView: nav.navigationBar)
    SideMenuManager.default.addScreenEdgePanGesturesToPresent(toView: nav.view, forMenu: .left)
 }else {
    SideMenuManager.default.addScreenEdgePanGesturesToPresent(toView: self.view, forMenu: .left)
}
```

consol log:

> Warning: Attempt to present <SideMenu.SideMenuNavigationController: 0x7fa75d019a00>  on <FinTech.IndexViewController: 0x7fa75e826800> which is already presenting (null)

**Comment by jonkykong** [comment]
How is this reproduced in the demo project?

**Comment by 22Round** [comment]
@jonkykong my approach is programmatic, please refer to attachment.
[I created small project to reproduce bug](https://github.com/jonkykong/SideMenu/files/3653252/SideMenuBug6.4.0.zip)

**Comment by 22Round** [comment]
@jonkykong any thoughts would be appreciated

**Comment by jonkykong** [comment]
Fixed in `6.4.3`

**Comment by 22Round** [comment]
@jonkykong thanks, fixed
